src/app/modules/auth/AuthController.ts

```diff
@@ -1,54 +1,53 @@
-const Boom = require('boom')
-const _ = require('lodash')
+import Boom from 'boom'
+import _ from 'lodash'
 
-const RegisterDTO = require('./dtos/RegisterDTO')
-const LoginDTO = require('./dtos/LoginDTO')
-const serviceLocator = require('../../services/serviceLocator')
-const log = require('../../util/logger')
+import {RegisterDTO} from './dtos/RegisterDTO'
+import {LoginDTO} from './dtos/LoginDTO'
+import serviceLocator from '../../services/serviceLocator'
+import {log} from '../../util/logger'
+import {AuthService} from '../../services/auth/AuthService'
 
 /**
  * TODO: Figure out better way to map errors in service layer to controller errors.
  * TODO: Prevent against CSRF attacks. @see https://www.acunetix.com/websitesecurity/csrf-attacks/
  */
 
-class AuthController {
-  static async register(request, reply) {
+export const AuthController = {
```

Comment:
You like creating an object over defining a class?

Reply:
I'm typically against creating a class with only static methods, because that's just a namespace, it has no member variables. Also classes with only static methods are still treated as classes by the type system which is not quite right. AND it fucks up the dependency injection.

But I'm not the most OO dude, so some of this might be personal prejudice.